climada/entity/exposures/base.py

```diff
@@ -461,20 +462,26 @@ def plot_raster(self, res=None, raster_res=None, save_tiff=None,
             ras_tiff.write(raster.astype(np.float32), 1)
             ras_tiff.close()
         # make plot
-        crs_epsg, _ = u_plot.get_transformation(self.crs)
-        xmin, ymin, xmax, ymax = self.gdf.longitude.min(), self.gdf.latitude.min(), \
-        self.gdf.longitude.max(), self.gdf.latitude.max()
+        proj_data, _ = u_plot.get_transformation(self.crs)
+        proj_plot = proj_data
```

Comment:
The problem with most of CLIMADA's plot functions is that they assume the same cartopy projection for the data as for the plot (axis). But the data might be given in a projection different from the axis projection. Here, I introduce two different variables for the two. Using this, the axis projection might shift the 0-longitude to a different location without any need to touch the data itself.

Reply:
Aaa now that makes sense!